tests/frontend/components/test_build_edit_section.py

```diff
@@ -0,0 +1,31 @@
+"""Test function build_edit_section."""
+
+import pytest
+from datadoc_model import model
+
+from datadoc.frontend.components.builders import build_edit_section
+from datadoc.frontend.fields.display_variables import OBLIGATORY_VARIABLES_METADATA
+
+NORSK_BOKMÅL = "nb"
+NORSK_NYNORSK = "nn"
+ENGLISH = "en"
+
+
+# empty input returns empty string/list
```

Comment:
We should either remove the comment or complete it and put it in the test docstring